### PR TITLE
CallToActionItem missing fields

### PR DESCRIPTION
**Pull request by depado**
Hey there 🎉 

Just noticed there were two missing fields in the `CallToActionItem` struct. 

See [this page](https://developers.facebook.com/docs/messenger-platform/thread-settings/persistent-menu) and especially the `menu_item` section for more details. 

`WebviewHeightRatio` was tested and works. Possible values (as described in the messenger docs) are : `compact`, `tall` or `full`

Thanks again for your great work, and I hope that will help ! 🎆

### CODE REVIEW

File: settings.go [hunk 1]
```diff
@@ -20,8 +20,10 @@ type CallToActionsSetting struct {
...
 type CallToActionsItem struct {
-	Type    string `json:"type,omitempty"`
-	Title   string `json:"title,omitempty"`
-	Payload string `json:"payload,omitempty"`
-	URL     string `json:"url,omitempty"`
+	Type               string `json:"type,omitempty"`
+	Title              string `json:"title,omitempty"`
+	Payload            string `json:"payload,omitempty"`
+	URL                string `json:"url,omitempty"`
+	WebviewHeightRatio string `json:"webview_height_ratio,omitempty"`
```
Comment (paked) [inline settings.go:44]: You mention in your comment about there being only three different values which are currently valid here. Personally, I think this warrants use of a constant.

I'd go for something like:

```go
const (
  WebviewCompact = iota
  WebviewFull
  WebviewTall
)
```

**Review by paked — CHANGES_REQUESTED** [review]

**Comment by paked** [comment]
Once we have this one change, I'll be more than happy to merge it in. 😀

File: settings.go (hunk 1 above)
Comment (depado) [reply settings.go:44]: Sure thing ! 🎈

**Comment by depado** [comment]
Hope that suits what you wanted, I quite didn't see how to use the `iota` mechanism in that case though.

**Comment by paked** [comment]
@Depado Yours was a much better way :) I must have been half asleep when I was thinking about the `iota` stuff!

**Review by paked — APPROVED** [review]

**Comment by depado** [comment]
Thanks a lot for reviewing my changes and helping me improve 🎉

**Comment by paked** [comment]
Thanks for the PR! (and working around my slightly-sleep-induced-crazy-review-requests)

**Pull request merged**